packages/core/src/button/base-button.element.scss

```diff
@@ -128,6 +128,10 @@
   --background: #{$cds-alias-status-success};
 }
 
+:host([status='warning']) {
+  --background: #{$cds-alias-status-warning-dark};
```

Comment:
To align with the spec similar to badge and alert warning states set the following for the warning button

```css
--background: #{$cds-alias-status-warning};
--color:  #{$cds-global-color-black};
```

Reply:
I actually went with `$cds-global-color-construction-900` instead of `$cds-global-color-black` because in `construction-900` matches with other buttons in dark theme, `black` was slightly different and felt out-of-place near the other buttons. Is that ok?

Reply:
Looks good to me. Nice work @astorije